_posts/blog/2018-11-02-el-topo-blindado.md

```diff
@@ -0,0 +1,59 @@
+---
+
+layout: post
+title: "El Topo Blindado: Centro de documentación de las organizaciones politico-militares argentinas"
+modified:
+categories: blog
+excerpt: ""
+tags: []
+author: nourry
+image:
+  feature:
+
+---
+
+## *El Topo Blindado: Centro de documentación de las organizaciones politico-militares argentinas.* 
+**eltopoblindado.com** 
+*Accessed 10 November 2018. </br> Reviewed by Daniel Nourry, University of Texas at Austin*
+
+A significant proportion of the twentieth century would see the people of Argentina, much like many if not all of the peoples of Latin America in the same period, governed by military dictatorships. Archive projects like the Archivo Histórico de la Policía Nacional de Guatemala are an important resource for understanding how these violent and repressive process functioned. As Kirsten Weld, author of *Paper Cadavers* - a book that records the events that led to the discovery and recovery of these important documents - tells us "People study history in order to participate in contemporary politics; we recover the past in order to look to the future. As such, documents, archives, and historical knowledge are more than just the building blocks of politics- they are themselves sites of contemporary political struggle" (3). In other words, the recovery, maintenance and promotion of archival records and sources is in itself a political act: a way of participating in the struggle. 
+
+This notion of actively participating in the struggle against oppression is precisely what the digital archive *El Topo Blindado: Centro de documentación de las organizaciones politico-militares argentinas* doubles up on. On the one hand, it makes manifest an archival process such as that described by Weld; and on the other, that which it collects, curates, archives and makes freely available are the documents, literature and records of groups that were formed precisely in order to critique these regimes of dictatorial violence in their day. 
+
+*Organizaciones Político-Militares* (OPM) is the broad name given to those groups whose political vision for a different and better social organization was wedded to their commitment to armed struggle as the vehicle through which to bring about these changes. In Argentina, the two most studied of these groups are the *PRT-ERP* and the *Montoneros*. They were, however, only two of the groups whose activities contributed to the broader phenomenon of OPM's. *El Topo Blindado: Centro de documentación de las organizaciones politico-militares argentinas* is the latest manifestation of an investment made by a collective of Argentine researchers (*El Topo Blindado*) - committed to the study of the role(s) played by OPM's in the recent and violent history of Argentina - to promote the deeper and more dynamic exploration of this broader phenomenon beyond the privileging of these two organizations and beyond the reduction of the activities of these kinds of groups to individual actions, pathologies, etc. 
```

Comment:
I find this paragraph confusing. I'm still unclear about basic facts about the project: what is it? Who made it? What does it contain? You can state this clearly and explicitly in this paragraph using simple and descriptive language: El Topo Blindado is a digital collection of historical records associated with organizations that resisted dictatorship in Argentina. The focus is on OPMs [define]. The project is created by [describe]. Simplifying your language will strengthen the discussion considerably.